packages/opentelemetry-core/src/context/propagation/B3Propagator.ts

```diff
@@ -26,6 +26,10 @@ import { getParentSpanContext, setExtractedSpanContext } from '../context';
 export const X_B3_TRACE_ID = 'x-b3-traceid';
 export const X_B3_SPAN_ID = 'x-b3-spanid';
 export const X_B3_SAMPLED = 'x-b3-sampled';
+export const X_B3_PARENT_SPAN_ID = 'x-b3-parentspanid';
+export const X_B3_FLAGS = 'x-b3-flags';
+export const PARENT_SPAN_ID_KEY = Context.createKey(X_B3_PARENT_SPAN_ID);
```

Comment:
The string passed into `createKey` is meant to be a human readable string and is only used for debugging purposes.

Reply:
Gotcha, thanks